### ISSUE TITLE
Moving panel causes panel to lose focus

### ISSUE THREAD

**Issue by leighmcculloch** [microsoft/vscode#111975]
- VSCode Version: 1.51.1
- OS Version: macOS 11

Steps to Reproduce:

1. Setup keybinding to change panel position, for e.g.:
   ```javascript
   {
       "key": "ctrl+a space",
       "command": "workbench.action.positionPanelRight",
       "when": "panelPosition == 'bottom'"
   },
   {
       "key": "ctrl+a space",
       "command": "workbench.action.positionPanelBottom",
       "when": "panelPosition == 'right'"
   },
   ```
2. While terminal in panel is focused, press `Ctrl+A Space`. The panel should move to the other position.
3. The terminal in the panel will no longer have focus. The only way to regain focus on the terminal is to use the mouse, or use some other keyboard shortcuts that give the terminal focus, such as create a new terminal or hide and reshow the terminal.

Does this issue occur when all extensions are disabled?: **Yes**

**Comment by sbatten** [comment]
@leighmcculloch This should be fixed, however using `ctrl+backtick` for me refocuses the terminal without having to toggle the terminal

**Comment by leighmcculloch** [comment]
@sbatten What version of VSCode do I need to upgrade to, to get this fix? I'm using 1.51.1.

**Comment by sbatten** [comment]
@leighmcculloch my apologies for my terrible wording. I meant to say this needs to be fixed, not that this has already been fixed.
